src/styles/blocks/watch.scss

```
@@ -0,0 +1,61 @@
+.watch {
+  position: relative;
+  background-color: white;
+  border: 15px solid rgb(255, 255, 255);
+  border-radius: 50%;
+
+  width: 500px;
+  height: 500px;
```

And maybe I would put the width in variables.
for example: $width-clock: 500px;
```suggestion
  width: $width-clock;
  height: $width-clock;
```